src/faq/assets/faq.css

```
@@ -0,0 +1,93 @@
+@tailwind base;
```

Stylelint (17.11.1)

[error] Unexpected unknown at-rule "`@tailwind`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)